client/src/index.js

```diff
@@ -4,13 +4,15 @@ import "./index.css";
 import App from "./components/App";
 import VerticalTabs from "./components/Dashboard/dashboard"
 import NavBar from "./components/navBar/navBar"
+import Flights from "./components/Dashboard/flights/flights"
 import reportWebVitals from "./reportWebVitals";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
     <NavBar/>
     <VerticalTabs />
+    <Flights />
```

Comment:
same as above, this should be in the dashboard component.

Reply:
when it is made that is

Reply:
Changed, thanks Christopher